script.js

```diff
@@ -0,0 +1,108 @@
+function colorSelected(event) {
+  const color = document.getElementsByClassName('color');
+  for (let index = 0; index < color.length; index += 1) {
+    color[index].classList.remove('selected');
+  }
+  event.target.classList.add('selected');
+}
+
+function fillPixel(event) {
+  const selectedColor = document.querySelector('.selected');
+  const eventTarget = event.target;
+  eventTarget.style.backgroundColor = selectedColor.style.backgroundColor;
+}
+
+function createPalette() {
+  const color = document.createElement('td');
+  color.className = 'color';
+  color.addEventListener('click', colorSelected);
+  return color;
+}
+
+function addPalette(quantidadeDeCores) {
+  const colorPalette = document.getElementById('color-palette');
+  for (let index = 0; index < quantidadeDeCores; index += 1) {
+    colorPalette.appendChild(createPalette());
+  }
+}
+
+function randonColor() {
+  const rgb = 'rgb(';
+  return rgb.concat(Math.ceil(Math.random() * 25) * 10, ', ',
+    Math.ceil(Math.random() * 25) * 10, ', ',
+    Math.ceil(Math.random() * 25) * 10, ')');
+}
+
+function addColor() {
+  const color = document.getElementsByClassName('color');
+  color[0].style.backgroundColor = 'black';
+  console.log(randonColor());
+  for (let index = 1; index < color.length; index += 1) {
+    color[index].style.backgroundColor = randonColor();
+  }
+}
+
+function createPixel() {
+  const pixel = document.createElement('td');
+  pixel.className = 'pixel';
+  pixel.addEventListener('click', fillPixel);
+  return pixel;
+}
+
+function quadroDePixels(linha, coluna) {
+  const pixelBoard = document.getElementById('pixel-board');
+  for (let indexC = 0; indexC < coluna; indexC += 1) {
```

Comment:
Felipe, o que significa esse C ? :) 
```suggestion
  for (let indexC = 0; indexC < coluna; indexC += 1) {
```